docs/api-docs/getting-started/webhooks/about-webhooks.md

```diff
@@ -90,28 +92,28 @@ When a webhook is triggered, BigCommerce will `POST` a light payload containing
  }
 ```
 
-A subsequent request can be made to [/orders/{id}](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/getanorder) to obtain full order details. 
+A request can then be made to [/orders/{id}](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/getanorder) to obtain full order details. 
 
 For more information on specific webhook events and their payloads, see [Webhook Events](https://developer.bigcommerce.com/api-docs/getting-started/webhooks/webhook-events).
 
-## Handling Callbacks
+## Handling callbacks
 
-To acknowledge the callback has been received without issue, the destination server must return an `HTTP 200` response -- any response outside the `200` range indicates the callback was not received. If this happens, the webhook service will use the [retry mechanism](#callback-retry-mechanism) described below.
+To acknowledge a callback has been received without issue, the destination server must return an `HTTP 200` response — no response, or any response outside the `200` range indicates the callback was not received. If this happens, the webhook service will use the [retry mechanism](#callback-retry-mechanism) described below.
 
 Need to set up a quick webhook destination URL for testing? See [Tools for Debugging and Testing Webhooks](#tools-for-debugging-and-testing-webhooks).
 
-## Callback Retry Mechanism
+## Callback retry mechanism
 
-The webhooks service will do its best to deliver events to the destination callback URI. It is best practice for the application to respond to the callback before taking any other action that would slow its response. If an app server responds to a webhook payload with anything other than a `2xx` response, or times out and indicates the payload has not been received, the following process will determine whether the destination URI gets blacklisted.
+The webhooks service will do its best to deliver events to the destination callback URI. It is best practice for the application to respond to the callback before taking any other action that would slow its response. Doing otherwise triggers BigCommerce's callback retry mechanism. 

```

Comment:
should this be callback URL, instead of callback URI?

Reply:
It can be used interchangeably. uniform resource identifier